cpu/kinetis_common/hwtimer_arch.c

```diff
@@ -27,8 +29,45 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+#define ENABLE_STATS (0)
+
+#if ENABLE_STATS
+/* Collect some statistics */
+struct {
+    unsigned int count_set_total;
+    unsigned int count_set_absolute;
+    unsigned int count_unset_total;
+    unsigned int count_isr_total;
+    unsigned int count_isr_rollover;
+    unsigned int count_isr_handler_call;
+    unsigned int count_isr_unset;
+} hwtimer_stats = {
+        .count_set_total = 0,
+        .count_set_absolute = 0,
+        .count_unset_total = 0,
+        .count_isr_total = 0,
+        .count_isr_rollover = 0,
+        .count_isr_handler_call = 0,
+        .count_isr_unset = 0,
+};
+
+#define HWTIMER_STATS_INC(x) (hwtimer_stats.x++)
```

Comment:
The identifier HWTIMER_STATS_INC sounds somehow confusing, I would write it small or/and insert a ..debug.. somewhere.